clib/src/keys.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2021 The Radicle Link Contributors
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use librad::{
+    keys::{IntoSecretKeyError, PublicKey, SecretKey},
+    profile::Profile,
+    signer::BoxedSigner,
+};
+use radicle_keystore::{
+    crypto::{Crypto, KdfParams, Pwhash, SecretBoxError},
+    file,
+    pinentry::Prompt,
+    FileStorage,
+    Keystore as _,
+};
+
+/// The filename for storing the secret key.
+pub const LIBRAD_KEY_FILE: &str = "librad.key";
```

Comment:
This pops up in a number of places in different codebases, is the intention that this be it's canonical home?

Reply:
Ya, that would be the intended purpose. Or we move it somewhere _even more_ canonical, and this lib uses it.